### ISSUE TITLE
Site Editor:  sidebar tab navigation issue

### ISSUE THREAD

**Issue by christospen** [WordPress/gutenberg#62535]
### Description

When trying to access patterns on editor there is an issue when on navigating using the tab key that prevents the list to scroll properly as a result the selected item is hidden because of the sticky elements (top or bottom of the sidebar)

### Step-by-step reproduction instructions

1. Go to theme editor
2. Go to patterns tab
3. Ensure that the the screen resolution is 1920x1080 and there are multiple items available to toggle the scroll bar.

### Screenshots, screen recording, code snippet

![Screenshot 2024-06-13 105003](https://github.com/WordPress/gutenberg/assets/14861144/509efc00-b7d4-42dd-bf53-4ad6fb96e388)
![Screenshot 2024-06-13 104958](https://github.com/WordPress/gutenberg/assets/14861144/b9e5987c-9098-42b8-b215-eaa7c0260c60)
![Screenshot 2024-06-13 104932](https://github.com/WordPress/gutenberg/assets/14861144/65219c41-832d-4550-a371-f063d9a0ed6d)
![Screenshot 2024-06-13 104908](https://github.com/WordPress/gutenberg/assets/14861144/d144564a-6796-4e73-81ab-be136f551480)

### Environment info

- Wordpress version: 6.5.4
- Gutenberg version: 17.7
- PHP 8.1.23
- Desktop Windows laptop (1920x1080 sceen resolution)
- Browser: Chrome (v 125.0.6422.142) and Firefox (121.0.1)
- Theme: Twenty Twenty Four

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by afercia** [comment]
Thanks @christospen for reporting this issue and sorry for the late reply.
This is a broader issue that occurs with any fixed or sticky header or footer in combination with scrollable elements and should be addressed across the entire editor. A while ago I created https://github.com/WordPress/gutenberg/issues/67959 to discuss a more generic approach, your thoughts and help are very welcome on that issue. If no objections, I'm going to close this issue in favor of https://github.com/WordPress/gutenberg/issues/67959
